ui/src/pages/Sign-up/sign-up.css

```diff
@@ -0,0 +1,21 @@
+.input-border {
+    border: 3px solid rgba(66, 153, 225, 0.5);
```

Comment:
tbh i'm a little confused? I thought we where using tailwind? it's probably me missing something so :) don't worry about me.

Reply:
there are some colors that are not supported in tailwind i can add them in config file then extend the colors but if am using it in one place i usally just put it in css if use it more than once then i extend it .